app/presenters/two_factor_login_options_presenter.rb

```diff
@@ -3,9 +3,10 @@ class TwoFactorLoginOptionsPresenter < TwoFactorAuthCode::GenericDeliveryPresent
 
   attr_reader :user
 
-  def initialize(user:, view:, service_provider:, aal3_required:, piv_cac_required:)
+  def initialize(user:, view:, context:, service_provider:, aal3_required:, piv_cac_required:)
```

Comment:
`context` is a very generic name.... can we call this `user_session_context`?

Reply:
> `context` is a very generic name.... can we call this `user_session_context`?

Yeah, makes sense. I'd meant to align it to the also-generic [`ApplicationController#context`](https://github.com/18F/identity-idp/blob/ba5e448af1c95563f3b9fe29644fe4ddbfff77fe/app/controllers/application_controller.rb#L84-L86), but given there's also a "view context" being passed here, seems worth differentiating the two.

Reply:
Updated in 8367c02.